notebooks/spark_basics/structured_streaming.ipynb

```diff
@@ -47,7 +47,7 @@
     "from pyspark import SparkContext\n",
     "from bigdl.util.common import *\n",
     "\n",
-    "sc = SparkContext('local')\n"
+    "sc = SparkContext('local[2]')\n"
```

Comment:
why 2 ?

Reply:
It's the min number of threads.
how do you think local[*] ?
It means to use all of logic cores.

Reply:
2 is a tricky number, if it's not specifically required to be 2,  * may be better

Reply:
OK, I will update it to [*] soon.